src/js/structs/CompositeState.js

```diff
@@ -103,6 +104,10 @@ class CompositeState {
   getNodesList() {
     return new NodesList({ items: this.data.slaves });
   }
+
+  getNodeMaster() {
+    return new Node(this.data.leader_info);
```

Comment:
We're missing this field in V1 😱 
There's a ticket for the mesos team https://issues.apache.org/jira/browse/MESOS-8078

Reply:
@dcos/frontend-foundation

Reply:
@nLight have you spoken to Vinod about getting this prioritized?

Reply:
Haven't spoken in person, I pinged him on slack, no reply though.

Reply:
Ok, so _there is_ a call for that.

`GET_MASTER` call gives the same info and it also should include region and zone info if that master is configured in a such way.

As @vinodkone told me, in V1 only leading master responds to request, so the info we get via `GET_MASTER` is equal to `leader_info`

Reply:
Please continue using `leader_info`, I will pick it up from three.